### ISSUE TITLE
Very slow run of python code in ipython vs. python

### ISSUE THREAD

**Issue by j9ac9k** [wuub/SublimeREPL#387]
I have been trying to send the current python file to iPython and I am noticing it takes ages (maybe 15 seconds?) to run.  Running the same code with just the standard python interpreter runs instantly.  I took the same code and loaded it up into Spyder and ran it with the interactive iPython interpreter and it ran very quickly.

I'm pretty new to sublime text (and sublimerepl) so I'm not sure what adequate troubleshooting steps I should take.

I have duplicated this behavior on Windows 7 x64 and Windows 8.1 x64 (I can try OS X later tonight). 

I should note, that I can also run the python code nearly instantly from the command line by running

```
ipython -i "<file path>\<file name.py>"
```

EDIT:

I should note that I can load ipython with sublimerepl very quickly (without passing any files to it).

**Comment by nafets33** [comment]
I am running into the same problem - did you find any solutions ?

**Comment by j9ac9k** [comment]
Nope, mostly just stopped trying to use subliemrepl, but that said there's been some commits to the repo here since I last tried.
